src/utilities/url-params.ts

```diff
@@ -77,5 +76,5 @@ export const DefaultUrlParams: QueryParams = {
   qaClear: undefined,
   testMigration: undefined,
 };
-
-export const urlParams: QueryParams = params;
+                                                    // allows use of ?demo for url
+export const urlParams: QueryParams = assign(params, { demo: typeof params.demo !== "undefined" });
```

Comment:
I'm sure somewhere in TypeScript/Babel/polyfill land Object.assign would work, but using lodash's assign means I don't have to think about the more complicated TypeScript/Babel/polyfill questions -- and it's less to type. 😉

Reply:
In case it wasn't clear, the real impetus for this was an update to query-string and @types/query-string after which the previous version no longer compiled.

Reply:
Yeah...I was trying to figure out that land today and I could not. Some further investigation has led me to conclude that we are *not* correctly transpiling `Object.assign` so the lodash call is necessary for ES5/IE11 support. Research also turned up `core-js`, which I think would add polyfills for most methods where we currently use `lodash`: https://github.com/zloirock/core-js

My preference would be to add polyfills and then just write ES6 code. It would clean up our code to remove `lodash` imports and keep us more standardized. 

This is probably a better discussion for future projects than this one, since we already use `lodash` here, but I'm still curious what you think.

Reply:
I think sorting out our ES5/polyfill situation is a good idea in any case. I suspect IE11 is off the table, however, given the extent to which we rely on SVG. I find `Object.assign` to be awfully verbose, and when you throw `Object.keys` and `Object.values` into the mix there's a lot to remember about what's available under what situations in addition to the verbosity. The nice thing about lodash is that it's short and sweet and always works and you don't have to think about it.

Reply:
It's true - lodash also adds utilities ES6 doesn't even support so we probably keep it around either way.